src/run.ts

```diff
@@ -55,6 +55,23 @@ export function run(
       showResults(results)
     })
 
+  program
+    .command('down-to')
+    .argument('<migration-name>')
+    .description('Migrates down to the specified migration name. Specify "NO_MIGRATIONS" to migrate all the way down.')
+    .action(async (name) => {
+      let results
+
+      if (name === 'NO_MIGRATIONS') {
+        console.log(`Migrating all the way down`)
+        results = await migrator.migrateTo(NO_MIGRATIONS)
```

Comment:
the `NO_MIGRATIONS` constant value is not a string, so it has to be imported directly from `kysely`.

Reply:
hm, just realized `kysely` is a dev dep, maybe we should just copy the `NO_MIGRATIONS` const value to this file and so it can remain as such:

```
export const NO_MIGRATIONS: NoMigrations = freeze({ __noMigrations__: true })
```

Reply:
Users will install `kysely` on their own, so I think the current code is okay

Reply:
I'm not 100% sure what `NO_MIGRATIONS` does, even though reading the doc :smile: 

https://github.com/kysely-org/kysely/blob/dd0319a8854e8e74c1b797354e1109b7bd95602e/src/migration/migrator.ts#L151-L158

However, it is officially supported by Kysely so I'll make this as is.